sockeye/model.py

```diff
@@ -154,26 +166,66 @@ def save_version(folder: str):
         with open(fname, "w") as out:
             out.write(__version__)
 
+    def _get_embed_weights(self) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, mx.sym.Symbol]:
+        """
+        Returns embedding parameters for source and target.
+
+        :return: Tuple of source and target parameter symbols.
+        """
+        assert isinstance(self.config.config_embed_source, encoder.EmbeddingConfig)
```

Comment:
Ugly python lacking proper type checking :-D However, are we expecting that this is not the case?

Reply:
we are not expecting this, but otherwise the IDE gives a type warning :/

Reply:
even if one used `self.config_embed_source = ... # type: encoder.EmbeddingConfig` in the `ModelConfig` class?